test/manual/cssPrefix.html

```diff
@@ -6,7 +6,7 @@
 <style>
 #el {font-size:20pt}
 </style>
-<script src="../../bower_components/jquery/jquery.js"></script>
+<script src="https://code.jquery.com/jquery-1.7.2.min.js"></script>
```

Comment:
It seems to be slow beacuse get a network. and is different case. I think that be better merged jquery.